### ISSUE TITLE
freegeoip.net has been shutdown

### ISSUE THREAD

**Issue by ptrojanowski** [benmurden/django-intl-tel-input#14]
```
{
  "0": "#################################################################################################################################",
  "1": "#                                                                                                                               #",
  "2": "# IMPORTANT - PLEASE UPDATE YOUR API ENDPOINT                                                                                   #",
  "3": "#                                                                                                                               #",
  "4": "# This API endpoint is deprecated and has now been shut down. To keep using the freegeoip API, please update your integration   #",
  "5": "# to use the new ipstack API endpoint, designed as a simple drop-in replacement.                                                #",
  "6": "# You will be required to create an account at https://ipstack.com and obtain an API access key.                                #",
  "7": "#                                                                                                                               #",
  "8": "# For more information on how to upgrade please visit our Github Tutorial at: https://github.com/apilayer/freegeoip#readme      #",
  "9": "#                                                                                                                               #",
  "a": "#################################################################################################################################"
}
```

**Comment by benmurden** [comment]
Thanks for the heads up on this. Intl-tel-input now gives https://ipinfo.io as an example. I will experiment with this, but it might be the case that leaving implementation up to the developer is the best option.

**Comment by ptrojanowski** [comment]
Leaving implementation up to the developer make sense.

**Comment by benmurden** [comment]
Resolved in v0.3.0.

Invoke the widget with `use_default_init=False` as in the following example, and provide your own initialization script after `{{ form.media.js }}`.

```python
class MyForm(forms.Form):
    tel_number = forms.CharField(widget=IntlTelInputWidget(
        use_default_init=False
    ))
```
